### PR TITLE
ml - added entities and repositories for courses

### LINKED ISSUE #21: Add a database table for the courses that get added to a personal schedule.
It should have fields:

- [x] id
- [x] enrollCd (enrollment code)
- [x] psId (personal schedule id)
- [x] yyyyq (the quarter in yyyyq format)
- [x] Add just the entity and repository class

### PR DESCRIPTION
**Pull request by Morgan6688**
# Overview

In this PR, I added entities and repositories for courses. 

# Issues Addressed

close #21 

# Details
image

### CODE REVIEW

**Review by andrewpengucsb — APPROVED** [review]
LGTM

File: src/main/java/edu/ucsb/cs156/courses/entities/Courses.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+@NoArgsConstructor
+@Builder
+@Entity(name = "courses")
+public class Courses {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+
+  private String enrollCd;
+  private String psld;
```
Comment (pconrad) [inline src/main/java/edu/ucsb/cs156/courses/entities/Courses.java:25]: Should this be psId (P S I D ) as in personal schedule identification) as opposed to psld ? (P S L D)

**Review by pconrad — CHANGES_REQUESTED** [review]
I saw one small naming issue that you may want to fix.

File: src/main/java/edu/ucsb/cs156/courses/entities/Courses.java (hunk 1 above)
Comment (andrewpengucsb) [reply src/main/java/edu/ucsb/cs156/courses/entities/Courses.java:25]: Good catch, didn't see this one when I reviewed!

**Review by andrewpengucsb — APPROVED** [review]
LGTM!

**Review by scsampath — APPROVED** [review]
Entity:
- Variables exist and are correct.

Repository:
- Repositor exists.

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**